docs/content/get-started/open-project.md

```diff
@@ -2,29 +2,38 @@
 
 There is a quick fix however:
 
-1. Open the FPS Starter Project in the Unity Editor located in `workers/unity`.
-1. In the Project window, navigate to **Assets** > **Fps** > **Art** > **Materials**.
-1. Right click on the `Source_Shaders` directory and select Reimport.
-
-<img src="{{assetRoot}}assets/shader-fix.png" style="margin: 0 auto; display: block;" />
+1. In your Unity Editor, open the FPS Starter Project; it's located in `workers/unity`.
+1. In your Editor's Project window, navigate to **Assets** > **Fps** > **Art** > **Materials**.
+1. Right-click on the `Source_Shaders` directory and select **Reimport**.
+<br/>
+<br/>
+![]({{assetRoot}}assets/shader-fix.png)<br/>
+_Unity Editor menu to reimport shaders_
 
 #### Bake Navmesh
-There is a [known issue]({{urlRoot}}/known-issues) in the Unity Editor regarding importing a navmesh. The navmseh for the `FPS-SimulatedPlayerCoordinator` is not imported correctly when you open the project for the first time. To fix this, you need to rebake the navmesh for this Scene. To do this:
+There is a [known issue]({{urlRoot}}/known-issues) in the Unity Editor regarding importing NavMesh. The Unity Editor does not import the NavMesh for the `FPS-SimulatedPlayerCoordinator` correctly when you open the project for the first time. To fix this, you need to rebake the NavMesh for this Scene. To do this:
 
 1. In the [Project window (Unity documentation)](https://docs.unity3d.com/Manual/ProjectView.html), open the `FPS-SimulatedPlayerCoordinator` Scene located at `Assets/Fps/Scenes`.
 1. In the [Hierarchy window (Unity documentation)](https://docs.unity3d.com/Manual/Hierarchy.html), click on the `FPS-Start_Large` object to see it in the [Inspector window (Unity documentation)](https://docs.unity3d.com/Manual/UsingTheInspector.html), and enable the object by clicking the checkbox next to its name.
 1. Open the **Navigation** window (Unity Editor menu: **Windows** > **AI** > **Navigation**).
 1. Select the **Bake** tab, and then the **Bake** button.
+1. You can verify that the NavMesh is baked correctly by navigating to **Assets** > **Fps** > **Scenes** > **FPS-SimulatedPlayerCoordinator** and checking that there's a NavMesh object listed. The NavMesh object is usually represented by the icon below. 
+<br/>
+<br/>
+![]({{assetRoot}}assets/navmesh-fixed.png)
+<br/>
+_NavMesh image showing it's baked correctly_
 
-You can verify that the NavMesh has been baked correctly by navigating to **Assets** > **Fps** > **Scenes** > **FPS-SimulatedPlayerCoordinator**, and checking that Unity displays the correct icon.
-<img src="{{assetRoot}}assets/navmesh-fixed.png" style="margin: 0 auto; display: block;" />
+> **TIP:** You may get a number of warnings displayed in the Unity Editor Console window. You can ignore the warnings at this stage; use the message icons on the right-hand side of the Console window to set it to display only info and error messages.
```

Comment:
there is actually a third bug that only affects some people (based on region). should we add this in this pr as well or in a separate one?

Reply:
Is there a ticket for it? Just thinking we need to get this in ASAP for the filming?!?

Reply:
if we don't get it in, we still need to at least mention it in the filming.

Reply:
OK - then let's get it in. Let me get this PR in and then we can add in a new one.